### PR TITLE
Danger id as context of Github status

### PR DESCRIPTION
**Pull request by thiagofelix**
This uses the danger_id to set the context of Github status. Should allow multiple status for multiple execution of danger cli.

See #570

### CODE REVIEW

File: spec/lib/danger/request_sources/github_spec.rb [hunk 1]
```diff
@@ -132,6 +132,14 @@
           @g.submit_pull_request_status!
         end.to raise_error("Couldn't find a commit to update its status".red)
       end
+
+      it "uses danger_id as context of status" do
+        options = hash_including(:context => "danger/special_context")
```
Comment (JuanitoFatas) [inline spec/lib/danger/request_sources/github_spec.rb]: TIL hash_including 🆒

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ## master
 
+* Use danger_id as context for Github status - Thiago Felix
```
Comment (JuanitoFatas) [inline CHANGELOG.md]: minor: Github should spelled as GitHub 😊

**Comment by JuanitoFatas** [comment]
Only two Rubocop offenses and we are good to go! 👍

File: CHANGELOG.md (hunk 2 above)
Comment (thiagofelix) [inline CHANGELOG.md]: Ops, working on the fix.

**Comment by thiagofelix** [comment]
@JuanitoFatas want me to fix the Rubocop offenses? They were already in the master branch when I forked.

File: lib/danger/request_sources/github.rb [hunk 3]
```diff
@@ -166,7 +167,7 @@ def submit_pull_request_status!(warnings: [], errors: [], details_url: [])
         begin
           client.create_status(ci_source.repo_slug, latest_pr_commit_ref, status, {
             description: message,
-            context: "danger/danger",
+            context: "danger/#{danger_id}",
```
Comment (orta) [inline lib/danger/request_sources/github.rb]: this is a really great idea @thiagofelix

File: spec/lib/danger/request_sources/github_spec.rb (hunk 1 above)
Comment (orta) [inline spec/lib/danger/request_sources/github_spec.rb]: me too

**Comment by orta** [comment]
OK, this just needs to rebase now 👍 then you are free to merge

**Comment by orta** [comment]
we do :

**Pull request merged**